python/servo/build_commands.py

```diff
@@ -404,10 +398,6 @@ def change_link_name(binary, old, new):
     install_name_tool(binary, '-change', old, f"@executable_path/{new}")
 
 
-def add_rpath_to_binary(binary, relative_path):
-    install_name_tool(binary, "-add_rpath", relative_path)
-
-
 def change_rpath_in_binary(binary, old, new):
```

Comment:
I believe change_rpath_in_binary can also be deleted as it is not used anywhere after the hombrew logic was removed.

Reply:
Good point! I've gone ahead and removed this function as well.